greetings-tycho/2.26.0/org.xtext.example.mydsl/pom.xml

```diff
@@ -33,36 +33,8 @@
 						<argument>rootPath=/${project.basedir}/..</argument>
 					</arguments>
 					<classpathScope>compile</classpathScope>
-					<includePluginDependencies>true</includePluginDependencies>
```

Comment:
TODO why was this introduced in the first place

Reply:
I guess because this way you don't have to specify such dependencies in the MANIFEST

Reply:
but these dependencies are in build.properties

Reply:
> but these dependencies are in build.properties

yes, but I think Tycho ignores them (it actually does, because without the required bundles the build fails)... see also https://github.com/eclipse/xtext/issues/1976#issuecomment-863252655 I don't know why they're there and who added them